pkg/parser/parser.go

```diff
@@ -148,6 +149,20 @@ func (p *Parser) parseTasks(tree *node.Node) ([]task.ParseableTaskLike, error) {
 
 			taskLike.SetID(p.NextTaskID())
 
+			// Emit a warning if the user tries to give multiple names to a single task
+			var hasQuickTaskName bool
+
+			if rn, ok := key.(*nameable.RegexNameable); ok {
+				if rn.FirstGroupOrDefault(treeItem.Name, "") != "" {
+					hasQuickTaskName = true
+				}
+			}
+
+			if taskLike.Name() != "" && hasQuickTaskName {
+				p.registerIssuef(api.Issue_WARNING, treeItem.Line, treeItem.Column,
+					"giving a task multiple names can be ambiguous")
```

Comment:
Let's maybe include `taskLike.Name()` in the warning message? Like `Task already has name defined as "%s". No need to use "*_task syntax"` or something similar?

Reply:
I don't think we need this since these messages are contextual, i.e. they are displayed right on the task that caused them.

Reply:
I don't think in this situation it will clear for a user why there are multiple names given for a task. IMO some human friendly message will better. This way user can map that there are tow places which define name:  'name: ' and '<name>_test' thing.

Reply:
Improved in https://github.com/cirruslabs/cirrus-cli/pull/429/commits/491304a751b19b19f2801894b19fba9eebcd9529.